### DISCUSSION TITLE
How to use RS232-2 port as terminal

### DISCUSSION THREAD

**Discussion by paulspann** [thehighnibble/thehighnibble.github.io#87] [IMSAI8080esp replica]
Hi,

Can anyone tell me the exact steps i need to do to connect an rs232 terminal to RS232-2 port???

Paul

**Comment by GrantMeStrength** [comment]
Did you get it working yet? If not, I can try and retrace my steps. Really, the web-based terminal is so much more convenient that I did it once, said "cool", and went back to the web version. :-)
